### PR TITLE
update iOS docs

### PR DESCRIPTION
**Pull request by atom2ueki**
- add notes for `AWSS3` v2.5.5
- add example for set and check bucket policy

### CODE REVIEW

File: docs/aws-sdk-for-iOS-with-minio.md [hunk 1]
```diff
@@ -59,5 +61,6 @@ for example if you running that eample project
 1. Run the xcode project on your phone or simulator
 2. Click upload button on screen
 <img src="/docs/screenshots/iOS-test-app.png" alt="screenshot" height="250">
-
-3. Check on `Minio Browser`, inside images bucket, there should be an image there called test.jpg, means you susccessful upload the image
+3. Set bucket policy prefix as `prefix` (becasue as you see the example code, remote name set as prefix_test.jpg) from `Minio Browser` or `Minio mc` to `images` bucket
```
Comment (NitishT) [inline docs/aws-sdk-for-iOS-with-minio.md]: It is not clear what is meant here, could you please rephrase to make it clearer?

File: docs/aws-sdk-for-iOS-with-minio.md [hunk 2]
```diff
@@ -59,5 +61,6 @@ for example if you running that eample project
...
+4. Check on `Minio Browser`, inside images bucket, there should be an image there called prefix_test.jpg, means you susccessful upload the image
```
Comment (NitishT) [inline docs/aws-sdk-for-iOS-with-minio.md]: `Minio Browser` here is the actual Minio browser or something else in the demo?

File: docs/aws-sdk-for-iOS-with-minio.md [hunk 3]
```diff
@@ -59,5 +61,6 @@ for example if you running that eample project
...
+5. Get remote url from iOS console or with this format 'host'/`bucket`/`filename with extension`, open in browser, you will see the image without any access deny
```
Comment (NitishT) [inline docs/aws-sdk-for-iOS-with-minio.md]: If you're talking about the Minio browser in the previous step, this is not required IMO.

File: docs/aws-sdk-for-iOS-with-minio.md (hunk 2 above)
Comment (atom2ueki) [inline docs/aws-sdk-for-iOS-with-minio.md]: Actual one

File: docs/aws-sdk-for-iOS-with-minio.md (hunk 3 above)
Comment (atom2ueki) [inline docs/aws-sdk-for-iOS-with-minio.md]: Sure, I will remove this part

**Comment by harshavardhana** [comment]
is this tested on your end @NitishT ?

**Pull request merged**